docs/openapi/components/schemas/credentials/FoodFacilityRegistrationCredential.yml

```diff
@@ -0,0 +1,178 @@
+$linkedData:
+  term: FFRCredential

```

Comment:
Can we expand the name to be the full name not the acronym? I believe that's our current rule for all credential names